src/shivangi/A21_PrintDigitFromString.java

```diff
@@ -0,0 +1,19 @@
+package shivangi;
+
+public class A21_PrintDigitFromString {
+	
+	void printDigitFromString() {
```

Comment:
the method should be parameterized.

Reply:
Okay mam, I tried to write basic level program.